src/app/home/home.component.html

```diff
@@ -41,6 +41,24 @@
                     <span id="mapCenterLabel" class="hover-content"></span>
                 </div>
             </div>
+            <mat-expansion-panel
+                id="legend"
+                class="legend"
+                data-html2canvas-ignore
```

Comment:
```suggestion

```

This ignore is not necessary. We won't be using html2canvas in this application.